src/Application.vala

```diff
@@ -21,10 +21,12 @@
 public class Installer.App : Gtk.Application {
     public const OptionEntry[] INSTALLER_OPTIONS = {
         { "test", 't', 0, OptionArg.NONE, out test_mode, "Non-destructive test mode", null},
+        { "load", 'l', 0, OptionArg.STRING, out load_view, "Force load a view", "VIEW"},
```

Comment:
I guess this should probably list all the valid options right?

Reply:
Yeah that's sane. I wonder if we can dynamically construct the options instead of just maintaining lists in multiple places?

Reply:
@danrabbit any further thoughts on this, it's hard to test the error view as-is. :stuck_out_tongue: